cms/admin/forms.py

```diff
@@ -179,6 +179,8 @@ def __init__(self, *args, **kwargs):
             self.fields['navigation_extenders'].widget = forms.Select({}, [('', "---------")] + menu_pool.get_menus_by_attribute("cms_enabled", True))
         if 'application_urls' in self.fields:
             self.fields['application_urls'].choices = [('', "---------")] + apphook_pool.get_apphooks()
+        self.fields['redirect'].widget.attrs['style'] = "width:221px"
+        self.fields['overwrite_url'].widget.attrs['style'] = "width:221px"
```

Comment:
What about specifying the widget for the two fields as AdminURLFieldWidget?
In this way we remove the direct width specification for the two fields, and get wider input fields for them.